### PR TITLE
AMQP changes (see commits)

### PR DESCRIPTION
**Pull request by clebertsuconic**

### CODE REVIEW

File: artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPMessage.java [hunk 1]
```diff
@@ -128,7 +130,7 @@ private void initalizeObjects() {
    }
 
    private Map getApplicationPropertiesMap() {
-
+      new Exception("trace").printStackTrace();
```
Comment (tabish121) [inline artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPMessage.java]: This seems and odd thing to have in production code, is it intentional ?

Comment (clebertsuconic) [inline artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPMessage.java]: oops.. not intended :) 

It's my usual debug...


I will have to fix this later tonight..

Comment (clebertsuconic) [inline artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPMessage.java]: @tabish121 thanks for catching it :)

**Comment by clebertsuconic** [comment]
Fixed

**Pull request closed without merging**